incubator/fluid-on-tke/Chart.yaml

```diff
@@ -14,11 +14,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.6.0
+version: 0.7.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: v0.6.0-e410a0b
+appVersion: v0.7.0-a0973270 
 icon: https://github.com/tkestack/charts/raw/main/icons/fluid-on-tke.jpg
```

Comment:
change icon address to: https://imgcache.qq.com/qcloud/tcloud_dtc/static/Industry_tke/5cd88288-e9c8-4aa7-9f42-5db67514b40f.jpg